shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/messaging/MessageReaderAdapter.java

```diff
@@ -88,6 +94,12 @@ private MetaStoreEvent map(ListenerEvent listenerEvent) {
       builder.partitionColumns(new ArrayList<>(insertTable.getPartitionKeyValues().keySet()));
       builder.partitionValues(new ArrayList<>(insertTable.getPartitionKeyValues().values()));
       break;
+    case ALTER_TABLE:
+      AlterTableEvent alterTable = (AlterTableEvent) listenerEvent;
+      if (alterTable.getTableLocation().equals(alterTable.getOldTableLocation())) {
```

Comment:
same here

Reply:
I am not sure. Can we have a table on a NULL location ?

Reply:
Also not sure but I guess if we use `Objects.equals` above then we might as well do the same here and then we're covered.

Reply:
Same response as above.